tower/src/buffer/service.rs

```diff
@@ -85,16 +84,15 @@ where
     }
 }
 
-impl<T, Request> Service<Request> for Buffer<T, Request>
+impl<Request, F, Resp, Error> Service<Request> for Buffer<Request, F>
```

Comment:
I know this is pre-existing, but I greatly prefer abbreviations for generics: `Req` & `Rsp`. Definitely more a personal taste thing than a blocker; but any time I see `Request` I assume there's an actual concrete type we're handling.

Reply:
sure, happy to change that --- i was just going with the existing naming. i agree that `Request` feels like a concrete name to me.